docker/docker.go

```diff
@@ -224,3 +229,14 @@ func IsContainerUpgradable(dClient *client.Client, imageCfg *container.Config, c
 	logrus.Debugf("[%s] Container %s is eligible for updgrade on host [%s]", plane, containerName, hostname)
 	return true, nil
 }
+
+func IsSupportedDockerVersion(info types.Info, K8sVersion string) (bool, error) {
+	for _, DockerVersion := range K8sDockerVersions[K8sVersion] {
+		DockerVersionRegexp := regexp.MustCompile("^" + DockerVersion)
+		if DockerVersionRegexp.MatchString(info.ServerVersion) {
```

Comment:
Instead of regex can't we use semver library https://godoc.org/github.com/blang/semver#Version.Compare